karate-demo/src/test/java/driver/demo/demo-01.feature

```diff
@@ -17,8 +18,53 @@ Scenario: try to login to github
   And input('#password', 'world')
   When submit().click("input[name=commit]")
   Then match html('#js-flash-container') contains 'Incorrect username or password.'
-  
+
   Given driver 'https://google.com'
   And input("input[name=q]", 'karate dsl')
   When submit().click("input[name=btnI]")
   Then waitForUrl('https://github.com/intuit/karate')
+
+Scenario: pass cookie from API call to UI call
```

Comment:
since I am used to using demo-01.feature for various live demos, can you move this into a new feature and java runner ? really appreciate that you took time to write a test 👍

Reply:
done! created new runner / feature for hybrid and reverted all -1 changes.